docs/IE_PLUGIN_DG/NewTransformation.md

```diff
@@ -1,15 +1,386 @@
 # Writing ngraph transformations {#new_ngraph_transformation}
 
-1. Code of such transformation MUST be directly in template plugin soruces.
-2. It must be mark with doxygen markeds like
+This guide contains all necessary information that could help you to start writing nGraph transformations.
 
+First of all before writing transformation make sure that there is no transformation with the same functionality in [transformation library].
+To start writing transformation it's good to know how [transformation library] is structured, how transformations are organized and where to put your transformation code.
 
-// ! [new_transformation:part1]
+Let's start from reviewing transformations library structure.
+Transformations library is independent from InferenceEngine target library and located in `inference-engine/src/transformations` directory.
```

Comment:
may be we need to put some information that Transformation library is actually `inference_engine_transformations`? How users can link this library to their plugins?

Reply:
I mentioned target name in description.